apps/services/auth-admin-api/src/app/modules/clients/clients.controller.ts

```diff
@@ -82,14 +82,14 @@ export class ClientsController {
     return await this.clientsService.update(client, id)
   }
 
-  /** Deletes a client by Id */
-  @Delete(':id')
+  /** Archive a client by Id */
+  @Put('archive/:id')
   @ApiCreatedResponse()
```

Comment:
Why not use @Delete ? If you don't want to use Delete since we aren't really deleting. Then I would use  @Patch instead of @Put.. since we aren't really doing a full update.

Reply:
Changed to @Delete